templates/base/agl-base-defaults.jinja2

```diff
@@ -21,13 +21,13 @@
 {%- if rootfs_type == 'nbd' %}
 {%- set deploy_to = "nbd" %}
 {%- set boot_commands = "nbd" %}
-{%- set nbdroot = nbdroot|default("core-image-minimal-" + yocto_machine + ".ext4.xz") %}
+{%- set nbdroot = rfs_image|default("core-image-minimal-" + yocto_machine + ".ext4.xz") %}
 {%- set nbdroot_url = nbdroot_url|default(baseurl(nbdroot)) %}
 {%- set nbdroot_compression = nbdroot_compression|default(nbdroot|get_extension) %}
 {%- set nbdinitrd = nbdinitrd|default("initramfs-netboot-image-" + yocto_machine +".ext4.gz") %}
 {%- set nbdinitrd_url = nbdinitrd_url|default(baseurl(nbdinitrd)) %}
 {%- elif rootfs_type == 'ramdisk' %}
-{%- set initrd = initrd|default("initramfs-boot-image-" + yocto_machine +".gz") %}
+{%- set initrd = rfs_image|default("agl-demo-platform-" + yocto_machine + ".ext4.xz") %}
```

Comment:
I don't get why you're changing the initrd from initramfs-* to agl-demo-platform-*.  Seems unrelated to this patch?

Reply:
In the first version of releng I was only trying to mimic the jobs posted on agl's lava as they are ... no care for consistency.

No that we can specify the image name, I thought it would be important that the default value is consistent between ramdisk and nbd

Do you think I should keep the previous value ?

Reply:
Yes, the defafult initrd image should be initramfs-boot-image.  It's only on QEMU that it needs to be overridden, so I would leave it that way.  I find it a bit confusing in this patch.